tutorial/README.md

```diff
@@ -0,0 +1,42 @@
+# Why Helm?
+
+[Helm](https://docs.helm.sh/) is often described as the Kubernetes application package manager. So, what does Helm give you over using Kubernetes directly?
+
+# Objectives
+
+These labs provide an insight on the advantages of using Helm over using Kubernetes directly through `kubectl`. When you complete all the labs, you'll:
+* Understand the core concepts of Helm
+* Understand the advantages of deployment using Helm over Kubernetes directly, looking at:
+  * Application management
+  * Updates
+  * Configuration
+  * Revision management
+  * Repositories and chart sharing
+
+# Prerequisites
+
+* Have a running Kubernetes cluster. See the [IBM Cloud Kubernetes Service](https://cloud.ibm.com/docs/containers/cs_tutorials.html#cs_cluster_tutorial) or [Kubernetes Getting Started Guide](https://kubernetes.io/docs/setup/) for details about creating a cluster.
+* Have Helm installed and initialized with the Kubernetes cluster. See [Installing Helm on IBM Cloud Kubernetes Service](Lab0/README.md) or the [Helm Quickstart Guide](https://docs.helm.sh/using_helm/#quickstart) for getting started with Helm.
+
+# Helm Overview
+
+Helm is a tool that streamlines installation and management of Kubernetes applications. It uses a packaging format called "charts", which are a collection of files that describe Kubernetes resources. It can run anywhere (laptop, CI/CD, etc.) and is available for various operating systems, like OSX, Linux and Windows.
+
+![helm-architecture](images/helm-architecture.png)
+
+It has a client-server architecture with the client called `helm` and the server called `Tiller`. The client is a CLI which users interact with to perform different operations like install/upgrade/delete etc. The client interacts with Tiller and the chart repository. Tiller interacts with the Kubernetes API server. It renders Helm template files into Kubernetes manifest files which it uses to perform operations on the Kubernetes cluster via the Kubernetes API. See the [Helm Architecture](https://docs.helm.sh/architecture/) for more details. 
+
+A [chart](https://docs.helm.sh/developing_charts) is organized as a collection of files inside of a directory where the directory name is the name of the chart. It contains template YAML files which facilitates providing configuration values at runtime and eliminates the need of modifying YAML files. These templates provide programming logic as they are based on the "Go template language", functions from the “Sprig” lib and other specialized functions.  
+The chart repository is a location where packaged charts can be stored and shared. This is akin to the image repository in Docker. Refer to [The Chart Repository Guide](https://github.com/helm/helm/blob/master/docs/chart_repository.md) for more details.
```

Comment:
I think you need a blank line before this paragraph- it renders kind of oddly w/o it.

Reply:
ok, done